server/datastore/mysql/locks_test.go

```diff
@@ -77,6 +78,13 @@ func testLocksLockUnlock(t *testing.T, ds *Datastore) {
 }
 
 func testLocksDBLocks(t *testing.T, ds *Datastore) {
+	row := ds.reader.QueryRowxContext(context.Background(), "SELECT VERSION()")
+	var version string
+	require.NoError(t, row.Scan(&version))
+	if strings.HasPrefix(version, "8.") {
+		t.Skip("#3626: DBLocks is not supported for mysql 8 yet.")
```

Comment:
https://github.com/fleetdm/fleet/issues/3626 (IMO not urgent to tackle now as part of this work.)